rental.rb

```diff
@@ -0,0 +1,23 @@
+# rental.rb
+
+#
+# Purpose: Rental class
+#
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+
+    # many to many relationship between book and person, using a join class (rental)
+    #
+    # setter for book (a rental can only belong to one book)
+    @book = book
+    book.rentals << self
+
+    # setter for person (a rental can only belong to one person)
+    @person = person
+    person.rentals << self
```

Comment:
- [ ] to make this relationship work Getters and Setters for @book and @person instance variables should be present

Reply:
Thank you. changes Done!